### DISCUSSION TITLE
AI deck camera window mode?

### DISCUSSION THREAD

**Discussion by noahzn** [bitcraze/discussions#453] [Q&A]
`The HM01B0 contains 324 x 324 pixel resolutions and supports a 324 x 244 window mode which can be readout at a maximum frame rate of 60FPS`.
What is window mode? In the fpv.py the resolution is 324x244. I tried to calibrate the camera, but I got the principal point `[156, 154]` It seems that the point should be the center of the resolution 324x324.

**Comment by knmcguire** [comment]
Hmm interesting, I'm afraid that we would need to investigate this further but currently we would not have an answer for this yet
